osu.Game/Screens/Select/FooterButton.cs

```diff
@@ -142,5 +161,34 @@ protected override bool OnKeyDown(KeyDownEvent e)
 
             return base.OnKeyDown(e);
         }
+
+        protected class ScalingDrawSizePreservingFillContainer : DrawSizePreservingFillContainer
```

Comment:
name this something more local, and make it private. it should only be used here.